Library/Formula/libgraphqlparser.rb

```diff
@@ -0,0 +1,39 @@
+require "open3"
+
+class Libgraphqlparser < Formula
+  desc "GraphQL query parser in C++ with C and C++ APIs"
+  homepage "https://github.com/graphql/libgraphqlparser"
+  url "https://github.com/graphql/libgraphqlparser/archive/v0.1.0.tar.gz"
+  sha256 "e4a12bba2f6c2db1b0528db5a0a3283fc5380ac5cadb8036793054cb7c9dd3ad"
+
+  depends_on "cmake" => :build
+  depends_on "bison" => :recommended
+  depends_on "flex"  => :recommended
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    bin.install "dump_json_ast"
```

Comment:
Since you aren't using `make install` you are probably missing headers.

Try to compile a simple program that uses libgraphqlparser with it installed.  E.g.

```sh
$ cat > main.c <<EOF
#include "graphqlparser/c/GraphQLParser.h"
#include "graphqlparser/c/GraphQLAstNode.h"

int main(int argc, char **argv) {
    struct GraphQLAstNode *node = graphql_parse_string("{ field }", NULL);
    if (node != NULL) {
        graphql_node_free(node);
        return 0;
    }
    return 1;
}
EOF
$ cc -lgraphqlparser main.c -o main
```